api/views/workflow.py

```diff
@@ -36,7 +36,7 @@ class ProviderViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = ProviderSerializer
 
     def get_queryset(self):
-        queryset = ShareUser.objects.exclude(robot='').exclude(long_title='')
+        queryset = ShareUser.objects.exclude(long_title='').exclude(favicon='')
```

Comment:
Ah fair enough, probably should have been more clear with the title. 

You're right about the OSF push user thing, I was looking at the remnants of when we were harvesting the preprint services and didn't actually check their permissions.